website/docs/reference/widgets/table/README.md

```diff
@@ -72,171 +83,144 @@ Controls how overflowing contents of the column are handled. When turned on, the
-| ---------------------- | ---------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- |
-| **selectedRow**        | It contains the data of the row selected by the user. It's an empty object if no row is selected.                       | `{{<table_name>.selectedRow}}`         |
-| **selectedRows**       | It contains an array of rows selected by the user when multi-select is enabled. It's \[null] if no row is selected.    | `{{<table_name>.selectedRows}}`        |
-| **triggeredRow**       | When a user interacts with an actionable item (like button) in a row, `triggeredRow` fetches the data of that column.           | `{{<table_name>.triggeredRow}}`        |
-| **isVisible**          | This property indicates whether the widget is visible or not.                                                                | `{{<table_name>.isVisible}}`           |
-| **sortOrder**          | This property helps you sort values.                                                                                         | `{{<table_name>.sortOrder}}`           |
-| **tableData**          | This property gets all the table data in JSON format.                                                                        | `{{<table_name>.tableData}}`           |
-| **selectedRowIndex**   | It gives the index of the row selected by the user. Not applicable when multiple rows are selected.                          | `{{<table_name>.selectedRowIndex}}`    |
-| **selectedRowIndices** | It gives you an array of the index of the rows selected by the user. Not applicable when multi-row selection is turned off.    | `{{<table_name>.selectedRowIndices`\}} |
-| **filteredTableData**  | Contains the data of the rows left after applying any selected filters, sort rule, or search terms.                  | `{{<table_name>.filteredTableData}}`   |
-| **pageNo**             | Contains the current page number that the user is on. APIs can use it for pagination                                         | `{{<table_name>.pageNo}}`              |
-| **pageSize**           | Contains the number of rows that can fit inside a page of the table. Changes along with the height & row height of the table | `{{<table_name>.pageSize}}`            |
-| **searchText**         | Contains the search text entered by the user in the Table                                                                    | `{{<table_name>.searchText}}`          |
+#### Client side search
 
-For the following examples of binding properties, consider a table called `Table_1` with the table data given below:
+Client side search controls whether the user can search the entire data set for their query, or only what's currently shown in the table widget.
```

Comment:
If we are adding this ensure you explain and give an example more than what is already there in the Properties table above.  If it will be added, let it right above Server side pagination in the structure shared

Reply:
I removed it for now -- I'd like to explain it better since it's not clear what it does, however I'll need to follow up with someone and update in a subsequent PR at some point I think. I spent a good while experimenting around with this setting weeks ago and was never able to get it to do anything meaningful.